database/index.js

```diff
@@ -35,7 +37,6 @@ const movieSchema = new mongoose.Schema(
 
 const personnelSchema = new mongoose.Schema(
   {
-    _id: 'ObjectId',
```

Comment:
What was the purpose of removing this schema reference?

Reply:
It was causing an error in my PUT and POST routes. Mongo wants a valid ObjectID for this, and removing it allowed it to create it for me (rather than having me create it as I post/put